src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3149,7 +3151,10 @@ void visitArgumentEvaluateAndUnsplit(int argumentIndex, bool assertsTrue, bool a
                 {
                     continue;
                 }
+
+                var argumentNoConversion = argumentsNoConversions[i];
                 VisitArgumentConversion(
+                    GetConversionOperandIfApplicable(arguments[i], argumentNoConversion),
                     argumentsNoConversions[i],
```

Comment:
>argumentsNoConversions[i] [](start = 20, length = 25)

`argumentNoConversion`

Reply:
re-activated

---
In reply to: [281850707](https://github.com/dotnet/roslyn/pull/35544#discussion_r281850707) [](ancestors = 281850707)